app/controllers/api/functions.php

```diff
@@ -1736,20 +1748,33 @@
                 $execution = Authorization::skip(fn () => $dbForProject->createDocument('executions', $execution));
             }
 
-            $queueForFunctions
-                ->setType('http')
-                ->setExecution($execution)
-                ->setFunction($function)
-                ->setBody($body)
-                ->setHeaders($headers)
-                ->setPath($path)
-                ->setMethod($method)
-                ->setJWT($jwt)
-                ->setProject($project)
-                ->setUser($user)
-                ->setParam('functionId', $function->getId())
-                ->setParam('executionId', $execution->getId())
-                ->trigger();
+            if(is_null($scheduledAt)) {
+                $queueForFunctions
+                    ->setType('http')
+                    ->setExecution($execution)
+                    ->setFunction($function)
+                    ->setBody($body)
+                    ->setHeaders($headers)
+                    ->setPath($path)
+                    ->setMethod($method)
+                    ->setJWT($jwt)
+                    ->setProject($project)
+                    ->setUser($user)
+                    ->setParam('functionId', $function->getId())
+                    ->setParam('executionId', $execution->getId())
+                    ->trigger();
+            } else {
+                $dbForConsole->createDocument('schedules', new Document([
+                    'region' => System::getEnv('_APP_REGION', 'default'),
+                    'resourceType' => 'execution',
+                    'resourceId' => $execution->getId(),
+                    'resourceInternalId' => $execution->getInternalId(),
+                    'resourceUpdatedAt' => DateTime::now(),
+                    'projectId' => $project->getId(),
+                    'schedule' => $scheduledAt,
+                    'active' => true,
```

Comment:
Compared to event triggered above, we are losing details here, like: body, headers, path, method, user.
JWT we shouldn't set here (it might expire), functions worker should generate it if we pass user.

I think we should add new parameter to `schedules` collection to store additional data like those, and just forward the mto function with execution.

Reply:
Once fixed, lets update test to ensure this. For example the function could receive body and log it - when we do getExecution, we can check it's logs and see if it includes the word.

Reply:
Added `metadata` column to hold this information in the `schedules` table